apps/web/src/pages/Applications/ApplicationSkillsPage/components/SkillTree.tsx

```diff
@@ -111,7 +111,12 @@ const SkillTree = ({
             skill={skill}
             availableExperiences={availableExperiences}
             trigger={
-              <Button type="button" color="secondary" mode="solid">
```

Comment:
I've been waffling on this for a while now, and since you're pointing it out, I'll change it back. It makes visual sense when in context with existing experiences, but I very much agree it's ambiguous when next to a warning.

Reply:
Okay this is done and pushed.